Remotion/Web/UnitTests/Core/Context/HttpContextStorageProviderTest.cs

```diff
@@ -76,19 +76,5 @@ public void FreeData_WithoutValue ()
       Assert.That (_provider.GetData ("Foo"), Is.Null);
       Assert.That (_testContext.Items.Contains ("Foo"), Is.False);
     }
-
-    [Test]
-    public void FallbackToCallContext_IfNoCurrentHttpContext ()
```

Comment:
Why did you drop thsi test?

Reply:
ah, I see. We can use PrivateInvoke.GetPrivateField(...) for this. It's not perfect but a good compromise for now.

Historical context information: back when we started with the SafeContextProvider and the IoC infrastructure, we couldn't do decorator implementations. Today, I might do the HttpContextStorageProvider as a decorator. Might still be a good refactoring but probably better suited to a separate task.